test/testmachinery/shoot/enable_disable_test.go

```diff
@@ -40,6 +42,10 @@ var _ = Describe("Shoot registry cache testing", func() {
 			common.AddOrUpdateRegistryCacheExtension(shoot, []v1alpha3.RegistryCache{
 				{Upstream: "ghcr.io", Volume: &v1alpha3.Volume{Size: &size}},
 			})
+			if common.IsVerticalPodAutoscalerEnabled(shoot) {
```

Comment:
Is it possible disable VPA in `BeforeSuite` and enable it again in `AfterSuite`? In this way we wouldn't need to duplicate the logic for disabling VPA in each and every test.